docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+services:
+  app:
+    build:
+      context: .
+
+    env_file:
+      - .env
+
+    ports:
+      - "8000:8000"
+
+    volumes:
+      - ./:/app
+      - my_media:/files/media
+
+    command: >
+      sh -c "/app/wait_for_db.sh &&
+            python manage.py migrate &&
```

Comment:
The `wait_for_db.sh` script is using `$POSTGRES_DB` as the host, which is incorrect. It should be `$POSTGRES_HOST` to properly connect to the database host. Please update this to ensure the script functions correctly.